test/helpers/cilium.go

```diff
@@ -231,8 +231,8 @@ func (s *SSHMeta) GetEndpoints() *CmdRes {
 // corresponding endpoint ID, and an error if the list of endpoints cannot be
 // retrieved via the Cilium CLI.
 func (s *SSHMeta) GetEndpointsIds() (map[string]string, error) {
-	// cilium endpoint list -o jsonpath='{range [*]}{@.container-name}{"="}{@.id}{"\n"}{end}'
-	filter := `{range [*]}{@.container-name}{"="}{@.id}{"\n"}{end}`
+	// cilium endpoint list -o jsonpath='{range [?(@.labels.orchestration-identity[0]!='reserved:health')]}{@.container-name}{"="}{@.id}{"\n"}{end}'
```

Comment:
Hmm, shouldn't we have a filter? Don't we eventually need to get all endpoints including the `reserved:health` ones?

Reply:
The filter is just made more specific to drop the cilium-health endpoint below. Most people writing tests don't care about the cilium-health endpoint, so it's easier if `GetEndpoints()` keeps that detail away. Otherwise we end up littering a bunch of extra cilium-health endpoint handling throughout the rest of the tests.

For tests that care about the cilium-health endpoint, they can fetch it separately.